### DISCUSSION TITLE
raphnet N64 to USB adapter

### DISCUSSION THREAD

**Discussion by y0shin** [cathery/sys-con#175] [Controller requests]
**Description of the controller**
[Link here.](https://www.raphnet-tech.com/products/n64_usb_adapter_gen3/index.php) A usb adapter for Nintendo 64 controllers. [Has a plugin for emulators](https://www.raphnet.net/programmation/mupen64plus-input-raphnetraw/index_en.php), which may be of help.

**Screenshot of the device**
Will post when recieved in the mail.

**Additional context**
Ability to set c-buttons as buttons or just right stick would be nice.

**Comment by Borrajo** [comment]
i made my own adapter raphnet ( in pc and rpi works! ).   im developer and will help with this!
